src/graphql/Raffle/Mutation.ts

```diff
@@ -0,0 +1,129 @@
+import { User, Context, generateToken, Auth} from '../../utils'
+import bcrypt from 'bcrypt'
+
+/**
+ * this function create user
+ * @param parent 
+ * @param args 
+ * @param ctx 
+ */
+async function createUser(parent: object, args: User, ctx: Context): Promise<User> {
```

Comment:
The types of the arguments should be deterministic and explicit. Just a plain "Object" is too generic and defeats the whole purpose.